refinery/core/models.py

```diff
@@ -1872,10 +1872,33 @@ def get_managed_group(self):
         except:
             return None
 
+    def duplicate_group_name_exists(self):
+        """
+        Checks if any other instance has duplicate name. Empty strings not
+        treated as duplicates
+
+        :returns: Boolean based on if a duplicate name exists
+        """
+        # Catches empty strings & removes trail/lead white spaces
+        if self.name:
```

Comment:
I'm not sure if we need this extra check here. This may have only been applicable to duplicate slug checking